auth/src/main/res/layout-land/fui_auth_method_picker_layout.xml

```diff
@@ -32,6 +32,16 @@
             android:id="@+id/btn_holder"
             style="@style/FirebaseUI.AuthMethodPicker.ButtonHolder" />
 
+        <TextView
```

Comment:
I believe you meant to put this out of the scroll view, right? If not, you can't put more than one view in a `ScrollView` so we'll need a LL.